content-src/components/DiscoveryStreamComponents/TopSites/_TopSites.scss

```diff
@@ -0,0 +1,10 @@
+.ds-top-sites {
+
+  .top-sites {
+    padding: 0;
```

Comment:
Since this property (and the one that follows are overriding stuff) that's defined elsewhere, I'd suggest a brief comment for each property override explaining why it's being overridden.